### PR TITLE
Update getDelegating function

### PR DESCRIPTION
**Pull request by lamcc21**
Ticket [#SQN-1227](https://onfinality.atlassian.net/browse/SQN-1227)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Changes

- Update .env files codegen
- Update getDelegating function

### CODE REVIEW

File: packages/network-clients/src/clients/networkClient.ts [hunk 1]
```diff
@@ -115,16 +115,15 @@ export class NetworkClient {
...
+    const eraNumber = currentEra.toNumber();
+    const ownStake = ownDelegation?.amount;
     const { totalDelegations } = delegator;
 
-    const sortedOwnStake = parseRawEraValue(ownStake, currentEra.toNumber());
-    const sortedTotalDelegations = parseRawEraValue(totalDelegations, currentEra.toNumber());
-    const sortedDelegating = sortedTotalDelegations.current.sub(sortedOwnStake.current);
-
-    return sortedDelegating;
+    const sortedOwnStake = ownStake ? parseRawEraValue(ownStake, eraNumber).after : BigNumber.from(0);
```
Comment (HZHAIX) [inline packages/network-clients/src/clients/networkClient.ts:124]: Should we use the `current` value?

Comment (lamcc21) [reply packages/network-clients/src/clients/networkClient.ts:124]: Right now the totalDelegations value is only updated when a new delegation is made. So we can either use the after value or set it when a new era occurs.

Screen Shot 2023-03-09 at 4 50 35 PM

Comment (lamcc21) [reply packages/network-clients/src/clients/networkClient.ts:124]: This useDelegating hook is only used in my-account page and in '/delegator/delegating'

Comment (HZHAIX) [reply packages/network-clients/src/clients/networkClient.ts:124]: Make sense

**Pull request merged**